modules/library/main/src/main/java/org/geotools/styling/DefaultResourceLocator.java

```diff
@@ -71,6 +72,23 @@ public URL locateResource(String uri) {
         return url;
     }
 
+    /**
+     * Variant to URL.getQuery() that won't balk at having # in the query (e.g., a color in a
+     * parametric SVG reference)
```

Comment:
I would put here an URL example, it would help understanding what the method does.